libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/model/DBUserLockType.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.oceanbase.tools.dbbrowser.model;
+
+/**
+ * @author yaobin
+ * @date 2023-09-27
+ * @since 4.2.3
+ */
+public enum DBUserLockType {
```

Comment:
if locktype, then boolean type will by a simple model, e.g.  accountLocked.

however, the db-browser module was designed as an general model for RDBMS.
we may follow oracle design, `account_status` , refer https://docs.oracle.com/en/database/oracle/oracle-database/19/refrn/DBA_USERS.html#GUID-309FCCB2-2E8D-4371-9FC5-7F3B10E2A8C0 .

Reply:
add DBUser in osc model which contains DBAccountLockType to indicate locked or not, and  DBAccountLockType can be easy transfer from db query result, and DBAccountLockType can convenient to show in log